community/logcheck/logcheck.cron.sample.d

```diff
@@ -0,0 +1,35 @@
+# /etc/cron.d/logcheck: crontab entries for the logcheck package
+ 
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+MAILTO=root
+ 
+@reboot         logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck -R; fi
+2 * * * *       logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck; fi
+
+# EOF
+
+
+## busybox crond reboot script [ /etc/periodic/reboot/logcheck ]
+
+#!/bin/sh
```

Comment:
Those lines are from the original sample by @ncopa - I think they may work with another cron (not busybox `crond`).

Let me know if you want if removed - my busybox version has been running ok for weeks now